tools/pylib/boutdata/data.py

```diff
@@ -599,11 +607,47 @@ def keys(self):
         """
         return self.varNames
 
-    def evolvingVariables(self):
-        """Return a list of names of time-evolving variables
 
+    @property
+    def shiftAngle(self):
+        """
+        The shift angle, calculated from zShift the first time it is used
+        """
+        if self._shiftAngle is None:
+            try:
+                nz = self["MZ"]
+            except KeyError:
+                try:
+                    nz = self.attributes("mz")
+                except KeyError:
+                    nz = self.attributes["mesh:nz"]
```

Comment:
I'm a bit confused about where this is coming from -- `mesh:nz` looks like an input file variable, but this class reads output files